sql/wsrep_applier.cc

```diff
@@ -127,6 +129,8 @@ void wsrep_store_error(const THD* const thd,
               dst.size(), dst.size() ? dst.data() : "(null)");
 }
 
+extern Wsrep_schema *wsrep_schema;
```

Comment:
This should not be necessary, as wsrep_schema is defined in wsrep_schema.h

Reply:
Corrected in 6092704619b